frontend/src/core/services/PocketUserService.js

```diff
@@ -194,6 +194,51 @@ class PocketUserService extends PocketBaseService {
     return `******${lastLetters}${emailProvider}`;
   }
 
+    /**
+   * Retrieve security questions from server.
+   *
+   * @return {Promise|Promise<{success:boolean, [data]: *}>}
+   * @async
+   */
+  async getSecurityQuestions() {
```

Comment:
Move this part to https://github.com/pokt-network/pocket-dashboard/blob/master/frontend/src/core/services/PocketSecurityQuestionsService.js, this is already a file for these data.